enrich/gpx/src/enrich.gpx.plugin.ts

```
@@ -180,11 +180,7 @@ async function enrichGpxData(
     tabularData = removeDuplicatePoints(tabularData)
   }
 
-  if (
-    filterDates &&
-    !isNaN(startDate.getTime()) &&
-    !isNaN(endDate.getTime())
-  ) {
+  if (filterDates && !isNaN(startDate.getTime()) && !isNaN(endDate.getTime())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)